modules/microshift-oc-adm-by-example-content.adoc

```diff
@@ -3,116 +3,30 @@
-  oc adm catalog mirror quay.io/my/image:latest file:///local/index
-  oc adm catalog mirror file:///local/index/my/image:latest my-airgapped-registry.com
-
-  # Configure a cluster to use a mirrored registry
-  oc apply -f manifests/imageContentSourcePolicy.yaml
-
-  # Edit the mirroring mappings and mirror with "oc image mirror" manually
-  oc adm catalog mirror --manifests-only quay.io/my/image:latest myregistry.com
-  oc image mirror -f manifests/mapping.txt
-
-  # Delete all ImageContentSourcePolicies generated by oc adm catalog mirror
-  oc delete imagecontentsourcepolicy -l operators.openshift.org/catalog=true
-----
-
-//== oc adm certificate approve
-//== oc adm certificate deny
-//== oc adm cordon
-//== oc adm create-bootstrap-project-template
-//== oc adm create-error-template
-//== oc adm create-login-template
-//== oc adm create-provider-selection-template
-//== oc adm drain
-//== oc adm groups add-users
-//== oc adm groups new
-//== oc adm groups prune
-//== oc adm groups remove-users
-//== oc adm groups sync
 
 == oc adm inspect
 Collect debugging data for a given resource
-//NOTE: This was hand-edited per QE in 4.13. This section is correct as is.
-.Example usage
-[source,bash,options="nowrap"]
-----
-  # Collect debugging data for the "microshift-apiserver"
-  oc adm inspect service/kubernetes
-
-  # Collect debugging data for the "microshift-apiserver" and "toptlvm-apiserver"
-  oc adm inspect service/kubernetes crd/logicalvolumes.topolvm.io
-
-  # Collect debugging data for services
-   oc adm inspect service
-
-  # Collect debugging data for all clusterversions
-  oc adm inspect service,crd
-----
-
-== oc adm migrate icsp
-Update imagecontentsourcepolicy file(s) to imagedigestmirrorset file(s).
-
-.Example usage
-[source,bash,options="nowrap"]
-----
-  # update the imagecontentsourcepolicy.yaml to new imagedigestmirrorset file under directory mydir
-  oc adm migrate icsp imagecontentsourcepolicy.yaml --dest-dir mydir
-----
-
-//== oc adm migrate template-instances
-//== oc adm must-gather
-//== oc adm new-project
-
-
-//== oc adm node-logs
-Display and filter node logs
 
 .Example usage
 [source,bash,options="nowrap"]
 ----
-  # Show kubelet logs from all masters
-  oc adm node-logs --role master -u kubelet
-
-  # See what logs are available in masters in /var/logs
-  oc adm node-logs --role master --path=/
-
-  # Display cron log file from all masters
-  oc adm node-logs --role master --path=cron
+  # Collect debugging data for the "openshift-apiserver" clusteroperator
+  oc adm inspect clusteroperator/openshift-apiserver
+  
+  # Collect debugging data for the "openshift-apiserver" and "kube-apiserver" clusteroperators
+  oc adm inspect clusteroperator/openshift-apiserver clusteroperator/kube-apiserver
+  
+  # Collect debugging data for all clusteroperators
+  oc adm inspect clusteroperator
+  
+  # Collect debugging data for all clusteroperators and clusterversions
+  oc adm inspect clusteroperators,clusterversions
```

Comment:
I tried these commands on a microshift cluster and i do not see them to be working, may be we should change what we are inspecting on.

Reply:
we have tested commands below and they work fine, may be we could incorporate some from here ? WDYT ?
oc adm inspect service/kubernetes
oc adm inspect crd/logicalvolumes.topolvm.io
oc adm inspect crd/rangeallocations.security.internal.openshift.io
oc adm inspect crd/routes.route.openshift.io
oc adm inspect node/<node_name>

Reply:
@kasturinarra Hello! Which ones did not work for you? Bc we only have the four oc adm commands here

Reply:
@kelbrown20 None of the command works for me. Does the same work for you ? Here is the output from microshift cluster when i copy paste the commands.
```
[redhat@dhcp-1-235-240 ~]$ oc adm inspect clusteroperator/openshift-apiserver
error: the server doesn't have a resource type "clusteroperator"
[redhat@dhcp-1-235-240 ~]$ oc adm inspect clusteroperator/openshift-apiserver clusteroperator/kube-apiserver
error: the server doesn't have a resource type "clusteroperator"
[redhat@dhcp-1-235-240 ~]$ oc adm inspect clusteroperator
error: the server doesn't have a resource type "clusteroperator"
[redhat@dhcp-1-235-240 ~]$ oc adm inspect clusteroperators,clusterversions
error: the server doesn't have a resource type "clusteroperators"
```

Reply:
Hmm, I know those are meant to be just examples where `oc adm inspect` can work with a bunch of other additions as well. But having the examples in the doc not work is not really great. @eggfoobar Did these work for you, or is this kinda what you were refering to with the `adm` commands referring to multiple apis so some commands won't work.

Reply:
Yeah, since we don't have the `clusteroperator` resources, we should remove them from the examples